src/main/java/org/mule/extension/http/internal/listener/HttpListenerProvider.java

```diff
@@ -217,6 +235,17 @@ private HttpServerConfiguration getServerConfiguration() {
     return builder.build();
   }
 
+  private void setReadTimeout(HttpServerConfiguration.Builder builder) {
+    Method method = getMethod(HttpServerConfiguration.Builder.class, "setReadTimeout", new Class[] {long.class});
+    if (method != null) {
+      try {
+        method.invoke(builder, connectionParams.getReadTimeout());
+      } catch (InvocationTargetException | IllegalAccessException e) {
+        throw new MuleRuntimeException(createStaticMessage("Exception while calling method by reflection"), e);
+      }
+    }
```

Comment:
Add something like this:
`else if (configuredValue != default) { log("this version of mule doesn't support to configure the readTimeout"); }`